### PR TITLE
Queue Workflows on pr creation

### PR DESCRIPTION
**Pull request by ramsessanchez**
Queue the workflows when dependabot creates a pr.

### CODE REVIEW

**Review by zengin — COMMENTED** [review]
Curious to know: what is the reason for these not being automatically queued? I don't see any filters in workflow description: https://github.com/microsoftgraph/msgraph-sdk-java/blob/dev/.github/workflows/codeql-analysis.yml#L9

File: .github/workflows/create-v1.0-pull-request.yml [hunk 1]
```diff
@@ -45,6 +45,30 @@ jobs:
...
         bin/hub pull-request -b "$BASE" -h "$GITHUB_REF" -m "$MESSAGE_TITLE" -m "$MESSAGE_BODY" -r "$REVIEWERS" -a "$ASSIGNEDTO" -l "$LABELS"
+    
+  queue-workflows:
+    needs: create-pull-request
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Queue Api Level Lint
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: "Checks the SDK only using APIs from the targeted API level"
```
Comment (baywet) [inline .github/workflows/create-v1.0-pull-request.yml:56]: > Required. The name or ID of the workflow to trigger and run. This is the name declared in the YAML, not the filename
[reference](https://github.com/benc-uk/workflow-dispatch#workflow)

**Review by baywet — CHANGES_REQUESTED** [review]
@zengin  the generation PR is created by a BOT (the default token from github) which doesn't trigger any other workflow (the default token can't trigger workflows, they did that to protect the platform and avoid cascading/infinite loops)

File: .github/workflows/create-v1.0-pull-request.yml (hunk 1 above)
Comment (ramsessanchez) [reply .github/workflows/create-v1.0-pull-request.yml:56]: That is the current name on the workflow

Comment (ramsessanchez) [reply .github/workflows/create-v1.0-pull-request.yml:56]: Should I include a name change to this workflow in this PR,  for the sake of being more concise?

Comment (baywet) [reply .github/workflows/create-v1.0-pull-request.yml:57]: sorry, I read that sentence the other way around...

**Review by baywet — APPROVED** [review]

**Pull request merged**